src/ast.scm

```diff
@@ -172,7 +172,8 @@
   (and (pair? e) (eq? (car e) 'ssavalue)))
 
 (define (symbol-like? e)
-  (or (symbol? e) (ssavalue? e)))
+  (or (and (symbol? e) (not (eq? e 'true)) (not (eq? e 'false)))
```

Comment:
Would it be worth adding `nothing` here as well? On 0.6-dev I'm able to assign `nothing`.